esp/services/esdl_svc_engine/esdl_binding.cpp

```diff
@@ -358,10 +358,20 @@ void EsdlServiceImpl::configureTargets(IPropertyTree *cfg, const char *service)
 
     if (target_cfg)
     {
-        IPropertyTree * customRequestTransform = nullptr;
+        Owned<IPropertyTree> serviceCrt;
         try
         {
-            customRequestTransform = target_cfg->queryPropTree("xsdl:CustomRequestTransform");
+            serviceCrt.set(target_cfg->queryPropTree("xsdl:CustomRequestTransform"));
+            if (!serviceCrt)
```

Comment:
Would it make sense to detect both the single transform and the list of transforms, at least to give feedback that the list will be ignored?